hw/ip/pinmux/data/pinmux_fpv_testplan.hjson

```diff
@@ -0,0 +1,119 @@
+{
+  name: "pinmux"
+  import_testplans: ["hw/dv/tools/testplans/fpv_csr_testplan.hjson"]
+  entries: [
+    {
+      name: InSel0_A
+      desc: "When register periph_insel sets to 0, the corresponding mio_to_periph_o should be 0."
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: InSel1_A
+      desc: "When register periph_insel sets to 1, the corresponding mio_to_periph_o should be 1."
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: InSelN_A
+      desc: '''When register periph_insel sets to any value between 2 and (2 + number of MioPads),
+      the corresponding mio_to_periph_o should be equal to the related mio_in_i value.'''
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: InSelOOB_A
+      desc: '''When register periph_insel sets to any value larger than (2 + number of MioPads),
+      the corresponding mio_to_periph_o should be 0.'''
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: MioToPeriphO_A
+      desc: '''This assertion checks that any changes from the mio_to_periph_o output is due to the
+      changes from either mio_in_i or register periph_insel.'''
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: OutSel0_A
+      desc: "When register mio_outsel sets to 0, the corresponding mio_out_o should be 0"
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: OutSel1_A
+      desc: "When register mio_outsel sets to 1, the corresponding mio_out_o should be 1."
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: OutSel2_A
+      desc: "When register mio_outsel sets to 2, the corresponding mio_out_o should be 0."
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: OutSelN_A
+      desc: '''When register mio_outsel sets to any value between 3 and (3 + Number of periph out),
+      the corresponding mio_out_o should be equal to the related periph_to_mio_i value.'''
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: OutSelOOB_A
+      desc: ''''When register mio_outsel sets to any value larger than (3 + Number of periph out),
```

Comment:
looks like you have too many ticks here... and further below you seem to have too many ticks as well ;).

Reply:
Thanks for the good catch Michael. Updated.